frontend/public/index.html

```diff
@@ -11,6 +11,7 @@
     <script src="https://js.stripe.com/v3/"></script>
     <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500">
 		<link rel="stylesheet" href="app.css">
+    <link rel='shortcut icon' type='image/x-icon' href='/frontend/src/images/favicon.ico' />
```

Comment:
@lorenzdesmedt you think this gonna work? The path for images in https://gitpay.me/frontend/src/images/favicon.ico is not valid.

Reply:
@alexanmtz yes, you are right, I have moved the favicon icon to the public folder and tested it (works now). I also had to resize the icon to remove the side borders.

![image](https://user-images.githubusercontent.com/34986102/58642600-f3322f80-82fd-11e9-8b88-490e9f338dce.png)

Reply:
@lorenzdesmedt awesome, great work, now we can merge it 🚀